go/signedexchange/certurl/parse.go

```diff
@@ -0,0 +1,77 @@
+package certurl
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+)
+
+func writeHead(w *bytes.Buffer, n int, size int) error {
+	for i := 0; i < size; i++ {
+		if err := w.WriteByte(byte(n >> (8 * uint(size-i-1)))); err != nil {
```

Comment:
Comment why you can't use https://golang.org/pkg/encoding/binary/#Write (size isn't necessarily a power of 2).